src/models/Block.js

```diff
@@ -12,4 +12,8 @@ export default class Block {
   valueOf() {
     return this.id
   }
+
+  toString() {
```

Comment:
I don't know if this is a good or bad idea. We have a test that asserts that blocks stringify to their ID. That test failed until I added this override.

Reply:
I'm surprised just `valueOf()` didn't work. Do you need both of them?

Reply:
This test only started failing after the dependency upgrades. Something to revisit before this is merged. I have bad feels about overriding `toString()`.